src/mappings/tokenDefinition.ts

```diff
@@ -0,0 +1,147 @@
+import { Address, BigInt } from '@graphprotocol/graph-ts'
```

Comment:
do we need these tokenDefinitions when we can directly call the contract to avoid as much hardcoding as possible?

Reply:
resolved